src/frontend/src/lib/constants/test-ids.constants.ts

```diff
@@ -1,6 +1,7 @@
 export const NAVIGATION_MENU_BUTTON = 'navigation-menu-button';
 export const NAVIGATION_MENU = 'navigation-menu';
 export const NAVIGATION_MENU_VIP_BUTTON = 'navigation-menu-vip-button';
+export const NAVIGATION_MENU_NETWORKS = 'navigation-menu-networks';
```

Comment:
This is not a navigation item, right?

```suggestion
export const NETWORKS_SWITCHER_DROPDOWN = 'networks-switcher-dropdown';
```